syncopy/datatype/methods/arithmetic.py

```diff
@@ -165,10 +171,64 @@ def _parse_input(obj1, obj2, operator):
         lgl = "Syncopy object, scalar or array-like"
         raise SPYTypeError(operand, varname="operand", expected=lgl)
 
-    return operand_dat, opres_type, operand_idxs
+    return baseObj, operand, operand_dat, opres_type, operand_idxs
+
+def _perform_computation(baseObj,
+                         operand,
+                         operand_dat,
+                         operand_idxs,
+                         opres_type,
+                         operator):
+    """
+    Coming soon...
+    """
+
+    # Prepare logging info in dictionary: we know that `baseObj` is definitely
+    # a Syncopy data object, operand may or may not be; account for this
+    if "BaseData" in str(operand.__class__.__mro__):
+        opSel = operand._selection
+    else:
+        opSel = None
+    log_dct = {"operator": operator,
+               "base": baseObj.__class__.__name__,
+               "base selection": baseObj._selection,
+               "operand": operand.__class__.__name__,
+               "operand selection": opSel}
+
+    # Create output object
+    out = baseObj.__class__(dimord=baseObj.dimord)
```

Comment:
Very neat! What happens in cases like `data1 = data1 + data2` with the underlying hdf5 container(s) of `data1`?

Reply:
Good catch!!! Works out of the box sequentially, but breaks in parallel.

Reply:
Correction - works in parallel, my tests were just wrong. Sorry for the confusion!